vscode/src/test/runTest.ts

```diff
@@ -0,0 +1,23 @@
+import * as path from 'path';
```

Comment:
The codes in this file are borrowed from the generated code in https://github.com/microsoft/vscode-generator-code.